vignettes/cis-analysis.Rmd

```diff
@@ -225,14 +225,3 @@ twas_z(weights, gwas$Z, R=R)
 ```{r}
 plot(weights)
 ```
```

Comment:
you should keep this here -- as i said, we will need a landing page for all analysis modules. let's keep this as a placeholder